src/v0/destinations/google_adwords_offline_conversions/networkHandler.js

```diff
@@ -174,6 +174,18 @@ const getConversionCustomVariableHashMap = (arrays) => {
   return hashMap;
 };
 
+/**
+ * Validates custom variable
+ * @param {*} customVariables
+ * @returns
+ */
+const isValidCustomVariables = (customVariables) => {
+  if (isDefinedAndNotNullAndNotEmpty(customVariables) && customVariables.length > 0) {
+    return !!(customVariables[0].from !== '' && customVariables[0].to !== '');
```

Comment:
Can you iterate over the entire customVariables to check this?

Reply:
updated